Modules/IO/Meta/test/itkMetaImageIOMetaDataTest.cxx

```diff
@@ -180,6 +180,17 @@ itkMetaImageIOMetaDataTest(int argc, char * argv[])
     std::string value("world");
     itk::EncapsulateMetaData<std::string>(dict, key, value);
   }
+
+  const auto maxSupportedStringSize = (MET_MAX_NUMBER_OF_FIELD_VALUES * sizeof(double)) - 1;
```

Comment:
Great that you also added a test! This commit should stay in ITK.

Reply:
Thanks @dzenanz. Interestingly, the test prints the entire string of 32767 chars to standard output, even when the test passes successfully, as it does:

    std::cout << "Key " << key << " found with expected value " << nativeValue << std::endl;

Which produces a lot of "xxx..." noise at https://dev.azure.com/itkrobotlinux/07610e7d-0753-4a9f-9f63-b3f6b749d57a/_apis/build/builds/4925/logs/11 

May I simply remove that line of code?

https://github.com/InsightSoftwareConsortium/ITK/blob/v5.1.2/Modules/IO/Meta/test/itkMetaImageIOMetaDataTest.cxx#L145

Reply:
Yes, you can remove printing the content in case of correct result.